modular-docs-manual/content/topics/using_text_snippets_or_text_fragments.adoc

```diff
@@ -1,26 +1,19 @@
-// Base the file name and the ID on the module title. For example:
-// * file name: my-concept-module-a.adoc
-// * ID: [id="my-concept-module-a_{context}"]
-// * Title: = My concept module A
-
-// The ID is an anchor that links to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
 [id="using_text_snippets_or_text_fragments_{context}"]
-// The `context` attribute enables module reuse. Every module ID includes a variable that sets the context, such as {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
-= Text Snippets or Text Fragments (Pseudo-modules)
-//In the title of concept modules, include nouns or noun phrases that are used in the body text. This helps readers and search engines find the information quickly.
-//Do not start the title of concept modules with a verb. See also _Wording of headings_ in _The IBM Style Guide_.
-
-[NOTE]
-The following standard is recommended when the documentation is being maintained without a Content Management System (CMS) capable of managing complex interrelations between modules.
-
-The use of reusable text snippet files (or text fragment files) is discouraged due to the complications that can arise due to the complexity they introduce.
 
-Snippet (fragment) file use should be limited to:
+= Text Snippets or Text Fragments
```

Comment:
```suggestion
= Text snippets or text fragments
```

Reply:
Thanks @ncbaratta . I make this title case on purpose to be parallel with the other headings in the guide, for now. We have an issue to fix all of these.

Reply:
Let's see if my comment triggers any other bites.  Maybe consider removing "text snippet" because Atom has a feature that uses the term "snippet". (I don't use Atom, so I can't elaborate.)  "Text fragment" throughout this section ought to be sufficiently clear.

Reply:
I agree, Mike. I removed 'snippet'.